app/assets/src/components/views/sample_page/index.js

```diff
@@ -0,0 +1,3 @@
+import SamplePage from "./SamplePage";
+
+export default SamplePage;
```

Comment:
I guess the question should have been here.
I have been calling this "Views", i.e. SampleView.

I would also like to understand: when do you use `samples_page` folder vs `SamplePage`?

Reply:
I think of "views" as things that can be reused in different pages, while "pages" are things that are only used on a particular page (roughly corresponds to an `.erb` file I guess). 

But I will do `SampleView` here for now. Since we have `PhyloTreeListView` which is very similar.

Regarding `samples_page` vs `SamplePage`, I think I made a mistake here. I think it should be `SamplePage` here, since it is really one big "component" with several related sub-components, as opposed to several distinct components. But it is a bit of a gray area.

Reply:
Yeah, we just need to sync on these concepts. `Views` were intended to be what you are calling `pages`.